iwf-sdk.yaml

```diff
@@ -1329,8 +1329,14 @@ components:
       properties:
         disableSystemSearchAttribute: # deprecated in favor of enabledExecutingStateExecutionIds
           type: boolean
-        enabledExecutingStateExecutionIds:
+        enabledExecutingStateExecutionIds: # deprecated in favor of executingStateIdMode
```

Comment:
It’s never being used. We can safely remove it

Reply:
Done now

Reply:
Should I keep it in `iwf.yaml`

Reply:
Doesn't seem to be used either. Removed